### PR TITLE
Added UI changes, warning text for SSH method, updated litmusctl commands and minor fixes

### PR DESCRIPTION
**Pull request by Saranya-jena**
## Proposed changes

- Added UI changes,
- Added warning text for SSH method, 
- Updated litmusctl commands and minor fixes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/public/locales/en/translation.yaml [hunk 1]
```diff
@@ -1378,7 +1378,7 @@ myhub:
     checkPreDefined: Check the pre-defined chaos scenario
     checkPreDefinedDesc: You can view the chaos scenario details here
   editPage:
-    edit: Edit the Hub
+    edit: Edit the ChaosHub
     click: Then click on the update button
     submit: Update MyHub
```
Comment (SarthakJain26) [inline litmus-portal/frontend/public/locales/en/translation.yaml:1383]: Should we update MyHub to ChaosHub?

File: litmus-portal/frontend/public/locales/en/translation.yaml [hunk 2]
```diff
@@ -1398,9 +1398,9 @@ customWorkflow:
...
     workflowDesc: Description
     chooseNamespace: Choose Namespace
-    firstChaos: Choose the hub
-    selectHub: Select the hub
-    public: Public Hub
+    firstChaos: Choose the ChaosHub
+    selectHub: Select the ChaosHub
+    public: Public ChaosHub
     configure: Construct your chaos scenario
     construct: Using experiments from MyHubs
```
Comment (SarthakJain26) [inline litmus-portal/frontend/public/locales/en/translation.yaml:1405]: same

File: litmus-portal/frontend/public/locales/en/translation.yaml [hunk 3]
```diff
@@ -1193,7 +1191,7 @@ createWorkflow:
 targets:
   options:
     upgrade: Upgrade
-    copyTargetID: Copy target ID
+    copyTargetID: Copy ChaosDelegate ID
```
Comment (amityt) [inline litmus-portal/frontend/public/locales/en/translation.yaml:1194]: `ChaosDelegate` --> `Chaos Delegate`

File: litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/index.tsx [hunk 4]
```diff
@@ -168,7 +168,7 @@ const WorkflowSettings = forwardRef((_, ref) => {
         workflowAction.setWorkflowManifest({ manifest: '' });
       }
       if ((value as ChooseWorkflowRadio).selected === 'C') {
-        setName('custom-chaos-scenario');
+        setName('custom');
```
Comment (amityt) [inline litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/index.tsx:171]: Please revert this back! 👀

**Review by amityt — CHANGES_REQUESTED** [review]

**Review by amityt — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**